### PR TITLE
WINDUP-1547: Included draft on troubleshooting the web console instal…

### PR DESCRIPTION
**Pull request by huffmanca**
WINDUP-1547: Included draft on troubleshooting the web console installation. This has been updated to include a commonly encountered issues section as well.

### CODE REVIEW

**Comment by huffmanca** [comment]
@jsight,

Adding you as the reviewer. This doesn't have the screenshots yet; however, the rest of the content should be complete. If we have any additional commonly encountered issues I would like to include them.

File: docs/topics/web-openshift-troubleshoot.adoc [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the {WebName} pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
+
+=== Pod Takes Longer than 600 seconds to become available 
+
+When examining any pod's logs the following error is seen.
+
+[source,options="nowrap",subs="+quotes"]
+----
+error: update acceptor rejected rhamt-web-console-executor-1: pods for rc 'rhamt/__POD_NAME__' took longer than 600 seconds to become available
```
Comment (jsight) [inline docs/topics/web-openshift-troubleshoot.adoc]: Isn't is likely that the cause of that is that the web console itself failed to start?

**Comment by jsight** [comment]
Should this have a section on the error message if there are no schedulable nodes for the requested parameters (eg, the CPU, storage, or memory requirements are too high)?

File: docs/topics/web-openshift-troubleshoot.adoc (hunk 1 above)
Comment (huffmanca) [reply docs/topics/web-openshift-troubleshoot.adoc]: I saw the same message on the web console pod as well, thus the attempt at making this one a bit more generic; however, if this is seen on the web console does it mean something different? Can this be enhanced further?

**Comment by huffmanca** [comment]
@jsight,

I've included a section on the out of quota limitation as well. I have one question regarding your prior comment, though.

File: docs/topics/web-openshift-troubleshoot.adoc (hunk 1 above)
Comment (huffmanca) [reply docs/topics/web-openshift-troubleshoot.adoc]: I reproduced this by creating three projects, and deploying the `empty-dir` template without changing the defaults. The first project started up successfully, but in both of the other two projects only the postgresql pod successfully started. 

For both of the other two projects the remaining pods, the web console and the executor, have `-deploy` in their title with an error. This is the error that was reported here. In this case I believe the deploy pod is the one that was failing, and it never was able to create the others?

If I attempt a redeployment of these pods it then proceeds to the insufficient resources error.

**Comment by jsight** [comment]
Perhaps you were waiting too long? I don't know how else you would see only
-deploy lines.

**Comment by huffmanca** [comment]
@jsight,

When I was attempting to reproduce this I created all three projects first, and then copied/pasted the template specifying a different project each time, attempting to spin up all pods in all projects simultaneously to recreate the error.

If we don't think this error is likely to be seen we could remove it, keeping the other section about resource contention?

Alternatively, I can demonstrate this over a bluejeans instance if necessary as well.

**Comment by jsight** [comment]
Just record a video of it happening, and maybe file a bug on OpenShift for the failure to create the pod entries?

**Comment by huffmanca** [comment]
@jsight,

I've updated the PR once again, moving the problematic section to the end of the frequently encountered issues and updating it to include both of the scenarios discussed. I'm still not fully happy with the way it's being represented, but let me know your thoughts on it.

**Comment by jsight** [comment]
I think it looks pretty good, tbh. I'm a little concerned about the difference between quota restrictions and openshift resource restrictions, but I'm not sure how the error messages differ. The one in there seems to be the issue where you have quota but the cluster can't currently service it (a surprisingly common problem, tbh).

Anyway, it can be tweaked later based on actual feedback.

**Comment by huffmanca** [comment]
Fair enough. I'll go ahead and merge this one.

Thanks for the continuous feedback and troubleshooting!

**Comment by jsight** [comment]
yw, and thanks for all your work on getting this together.

**Pull request merged**